zh/enable-tls-for-mysql-client.md

```diff
@@ -606,12 +606,6 @@ summary: 在 Kubernetes 上如何为 TiDB 集群的 MySQL 客户端开启 TLS。
             cluster: ${cluster_name}
             clusterNamespace: ${namespace}
             sendCredToTikv: true
```

Comment:
Please update https://github.com/pingcap/docs-tidb-operator/blob/0dde3d9f40bcde85178ed8ed9a82f6d1c275dae1/zh/enable-tls-for-mysql-client.md?plain=1#L394-L395

Reply:
https://github.com/pingcap/docs-tidb-operator/pull/2615/commits/9aea0fbe726a549f674632269c5fda7646c6e939